src/Views/TryInstallView.vala

```diff
@@ -155,7 +155,22 @@ public class Installer.TryInstallView : AbstractInstallerView {
             }
         });
 
+        custom_button.clicked.connect (() => {
```

Comment:
This looks like a copy-paste of the block above. Am I missing something?

Reply:
@cassidyjames Feel free to do whatever you want with these PRs. The diff between Pop!_OS's installer and elementary's installer is so large now, even from the time of submitting these PRs, that there isn't much point in me spending time on them, when it'd be easier for elementary to rebase themselves on Pop's repo and make whatever changes they want instead.